website/frontend/templates/docs/plugin-api.html

```diff
@@ -71,6 +73,30 @@ <h1 id="file_formats">File Formats</h1>
 
       register_format(MyFile)</pre>
 
+	  <p>
+		Variables explanation:
+		<li>"PLUGIN_DESCRIPTION" should be as simple as possible, while still describing the main function</li> 
+		<li>"PLUGIN_VERSION" should be filled with the version of Plugin</li>
+		<li>"PLUGIN_API_VERSIONS" should be set to the version of Picard this plugin to run with.</li>
+		<li>(if the plugin has a license, add "PLUGIN_LICENSE" and "PLUGIN_LICENSE_URL")</li> 
```

Comment:
In fact, we (almost) require an explicit licence now, so i'd remove 'if the plugin has a licence'